bootstrap.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+if test ! $(which brew); then
+    echo "ERROR: It looks like you need to install Homebrew first."
```

Comment:
This means we are only compatible with MacOS from this point on, at least from an automation POV. This might be a problem for NRCan maintaining it.

Reply:
Yeah, this will work for us but it's not very portable. 

Since it's only used to install Python 3 and Mongo, this shouldn't be a deal-breaker for us. The stuff in the Makefiles *should be* platform-agnostic.